dictdiffer/utils.py

```diff
@@ -270,6 +270,17 @@ def are_different(first, second, tolerance):
         return not (first_is_nan and second_is_nan)
     elif isinstance(first, num_types) and isinstance(second, num_types):
         # two numerical values are compared with tolerance
-        return abs(first-second) > tolerance * max(abs(first), abs(second))
+        if tolerance is not None:
+            rel_diff = (
+                abs(first-second) > tolerance * max(abs(first), abs(second))
+            )
+        else:
+            rel_diff = True
+        # two numerical values are compared with absolute_tolerance
+        if absolute_tolerance is not None:
+            abs_diff = (abs(first-second) > absolute_tolerance)
+        else:
+            abs_diff = True
+        return rel_diff and abs_diff
```

Comment:
I just realized that what we want already exists in `math`, so we could just write this:

```suggestion
    return not math.isclose(
            first,
            second,
            rel_tol=tolerance or 0,
            abs_tol=absolute_tolerance or 0,
        )
```
It is easier to understand, already documented and probably more efficient.

Reply:
This is much better! I'll close my PR in favour of this.